examples/v1beta1/pipelineruns/alpha/pipelinerun-array-resluts-indexing.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipelinerun-array-indexing-results
+spec:
+  pipelineSpec:
+    tasks:
+      - name: task1
+        taskSpec:
+          results:
+            - name: array-results
+              description: The array resluts
```

Comment:
there's a typo here and in the name of this file

Reply:
isn't specifying `type` necessary here?

Reply:
> there's a typo here and in the name of this file

Thank you!

> isn't specifying type necessary here?
Yes, I will add this, thanks for catching! It is not failing in this pr because the validation we discussed is implemented in this pr https://github.com/tektoncd/pipeline/pull/4920

Reply:
filename still needs to be fixed

Reply:
Oh sorry my bad!